src/Generator.php

```diff
@@ -24,7 +25,15 @@ public static function generateDocs()
             $swagger = \Swagger\scan($appDir, ['exclude' => $excludeDirs]);
 
             if (config('l5-swagger.paths.base') !== null) {
-                $swagger->basePath = config('l5-swagger.paths.base');
+                if (config('swagger_version') === '3.0') {
+                    $swagger->servers = [
+                        new Server(['url' => config('l5-swagger.paths.base')]),
+                    ];
+                }
+
+                if (config('swagger_version') !== '3.0') {
```

Comment:
Why not just  `else`?

Please use this for version comparison:
```php 
version_compare(config('l5-swagger.swagger_version'), '3.0', '>=')
```

Reply:
Code climate is correct here :) 

`$swagger->basePath = config('l5-swagger.paths.base');` should go first and the the IF statement :)

Reply:
I don't think so. If we have Swagger 3, we need to set servers propery, otherwise basePath property. If I will assign basePath propery first, then I will need to unset it in IF statement. Are you really sure this really means "simplify"? :-)

Reply:
Maby you forgot to push them?

Reply:
No, please check current PR diff:
https://github.com/DarkaOnLine/L5-Swagger/pull/146/files

Reply:
It should be 2 `if` statements as you said and did here: https://github.com/DarkaOnLine/L5-Swagger/pull/146/commits/c16f5495ec5f27eb15aee586934e3bd2dd911ebc

Reply:
Sorry for misunderstanding, replaced and pushed. Please review.